modules/ROOT/pages/audit-logging.adoc

```diff
@@ -1362,510 +1383,538 @@ properties: name, isPublic
 |Public portal page
 |Page path
 |Public portal
 |Create
-|subaction: none +
-properties: portal ID, page ID
+|Subaction: None +
+Properties:
+
+* Portal ID
+* Page ID
 
 |Delete a page
 |Public portal page
 |Page path
 |Public portal
 |Delete
-|subaction: none +
-properties: portal ID, page ID
+|Subaction: None +
+Properties:
+
+* Portal ID
+* Page ID
 
 |Update a page
 |Public portal page
 |Page path
 |Public portal
 |Update
-|subaction: none +
-properties: portal ID, page ID
+|Subaction: None +
+Properties:
+
+* Portal ID
+* Page ID
 
 |Create a portal
 |Public portal
 |Organization ID
 |N/A
 |Create
-|subaction: none +
-properties: project ID, created by ID
+|Subaction: None +
+Properties:
+
+* Project ID
+* Created by ID
 
 |Publish a portal
 |Public portal
 |Organization ID
 |N/A
 |Publish
-|subaction: none +
-properties: project ID, created by ID
+|Subaction: None +
+Properties:
+
+* Project ID
+* Created by ID
 
 |Delete a portal
 |Public portal
 |Organization ID
 |N/A
 |Delete
-|subaction: none +
-properties: project ID, created by ID
+|Subaction: None +
+Properties:
+
+* Project ID
+* Created by ID
 
 |Update a portal
 |Public portal
 |Organization ID
 |N/A
 |Update
-|subaction: none +
-properties: project ID, created by ID
+|Subaction: None +
+
+Properties:
+
+* Project ID
+* Created by ID
 |===
 
-==== Asset Reviews
+=== Asset Reviews
 
 [%header,cols="25a,15a,15a,15a,15a,30a"]
 |===
 |User Action
-|Obj Type
+|Object Type
 |Object
 |Parent
 |Action
 |Payload
 
-|Create a comment
+|Create a Comment
 |Asset portal review comment
 |Comment ID
 |Asset portal review
 |Create
-|subaction: none +
-properties: comment
+|Subaction: None +
+Properties: Comment
 
 |Delete a comment
 |Asset portal review comment
 |Comment ID
 |Asset portal review
 |Delete
-|subaction: none +
-properties: comment
+|Subaction: None +
+Properties: Comment
 
 |Update a comment
 |Asset portal review comment
 |Comment ID
 |Asset portal review
 |Update
-|subaction: none +
-properties: comment
+|Subaction: None +
+Properties: Comment
 
 |Create a review
 |Asset portal review
 |Review ID
 |Asset
 |Create
-|subaction: none +
-properties: review object
+|Subaction: None +
+Properties: Review object
 
 |Delete a review
 |Asset portal review
 |Review ID
 |Asset
 |Delete
-|subaction: none +
-properties: review object
+|Subaction: None +
+Properties: Review object
 
 |Update a review
 |Asset portal review
 |Review ID
 |Asset
 |Update
-|subaction: none +
-properties: review object
+|Subaction: None +
+Properties: Review object
 |===
 
-=== Runtime Manager
+== Runtime Manager
 
 
 [%header,cols="25a,15a,15a,15a,15a,30a"]
 |===
-|User action
-|Obj Type
+|User Action
+|Object Type
 |Object
 |Parent
 |Action
 |Payload
-|Create Application
+|Create application
 |Application
 |AppID
-|n/a
+|N/A
 |Create
-|subaction : none +
-properties : application metadata
-|Start Application
+|Subaction: None +
+Properties: Application metadata
+|Start application
 |Application
 |AppID
-|n/a
+|N/A
```

Comment:
Is the "Payload" cell empty on purpose for this one entry?

Reply:
I'd be shocked if it were. I will reach out to the appropriate party to ask. Good catch!!